tasks/community-modules.yml

```diff
@@ -5,19 +5,12 @@
     dest: "{{ odoo_role_odoo_modules_path }}/requirements.txt"
     owner: "{{ odoo_role_odoo_user }}"
     group: "{{ odoo_role_odoo_group }}"
+  tags:
+    - mods
 
 - name: Deploy community roles with pip
   pip:
     requirements: "{{ odoo_role_odoo_modules_path }}/requirements.txt"
     virtualenv: "{{ odoo_role_odoo_venv_path }}"
-
-- name: Update Odoo modules list
-  become: yes
-  become_user: "{{ odoo_role_odoo_user }}"
-  command: "{{ odoo_role_odoo_python_path }} {{ odoo_role_odoo_bin_path }} -c {{ odoo_role_odoo_config_path }}/odoo.conf -d {{ odoo_role_odoo_db_name }} --update all --stop-after-init --without-demo=all --logfile=/dev/stdout --log-level=warn"
```

Comment:
It seems that it's not needed [anymore](https://github.com/odoo/odoo/issues/11802#issuecomment-387314916)